common/religion/doctrines/wc_doctrines_special.txt

```diff
@@ -212,3 +212,17 @@ doctrine_legions_puppet = {
 		}
 	}
 }
+doctrine_celestial_communion = {
+	group = "special"
+	
+	is_available_on_create = {
+		has_doctrine = special_doctrine_celestial_communion # Keeps on creating a new faith
+	}
+	
+	special_doctrine_celestial_communion = {
+		parameters = {
+			special_doctrine_celestial_communion_active = yes 
+			hostility_override_special_doctrine_celestial_communion = 1
+		}
+	}
+}
```

Comment:
Maybe make the existing tenet of Pacifism work the same way?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/149783033-42717fce-6f5a-4d57-984d-e37df3d7d1bd.png)

</details>